rdt-app/build.gradle

```diff
@@ -18,12 +18,12 @@ buildscript {
 
 allprojects {
     repositories {
+        mavenLocal()
```

Comment:
Maven local should be the last repository used, so that its has the least preference

Reply:
Done.